### PR TITLE
[Docs] Script to download openrpc specs

### PR DESCRIPTION
**Pull request by ronny-mysten**
## Description 

Downloading the openrpc specs from github on doc site build to ensure always using latest.

## Test Plan 

Locally

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: docs/site/src/utils/getopenrpcspecs.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+const axios = require('axios');
+const fs = require('fs');
+const path = require('path');
+
+// Download Mainnet OpenRPC spec
```
Comment (stefan-mysten) [inline docs/site/src/utils/getopenrpcspecs.js:8]: I am not a JS expert, but I think it'd be better to have one function that is called with different parameters than copy pasting the code three times. 
```suggestion
function download_spec(branch) {
axios({
    method: "get",
    url: "https://raw.githubusercontent.com/MystenLabs/sui/${branch}/crates/sui-open-rpc/spec/openrpc.json",
    responseType: "blob"
}).then((res) => {

    if (fs.existsSync(path.join(__dirname, "../open-spec/${branch}/openrpc_backup.json"))){
        fs.unlink(path.join(__dirname, "../open-spec/${branch}/openrpc_backup.json"), (err) => {
            if (err) {
                return console.log(err);
            }
            console.log("Deleted ${branch} backup spec.")
        } )
    } else {
        console.log("Backup file for ${branch} does not exist.")
    }
    if (fs.existsSync(path.join(__dirname, "../open-spec/${branch}/openrpc.json"))){
        fs.renameSync(path.join(__dirname, "../open-spec/${branch}/openrpc.json"), path.join(__dirname, "../open-spec/${branch}/openrpc_backup.json"));
    }

    fs.writeFileSync(path.join(__dirname, "../open-spec/${branch}/openrpc.json"), res.data, 'utf8');
}).catch(err => {
    console.log("Error downloading ${branch} openrpc spec.");
    console.error(err);
})
// Download Mainnet OpenRPC spec
download_spec("mainnet");
// Download Devnet OpenRPC spec
download_spec("devnet");
// Download Testnet OpenRPC spec
download_spec("testnet");
```

**Review by stefan-mysten — APPROVED** [review]
See comment regarding using a function instead of copy pasting the dwl code three times

File: docs/site/src/utils/getopenrpcspecs.js (hunk 1 above)
Comment (ronny-mysten) [reply docs/site/src/utils/getopenrpcspecs.js:31]: Thought the same but wanted to be verbose because it was small. Will update.

Comment (stefan-mysten) [reply docs/site/src/utils/getopenrpcspecs.js:31]: Ultimately, it's up to you. Having a function will lower the chances of having bugs that might need to be fixed in 3 places.

Comment (ronny-mysten) [reply docs/site/src/utils/getopenrpcspecs.js]: Yep. I updated. And the vercel preview worked so merging. Thanks!

Comment (stefan-mysten) [reply docs/site/src/utils/getopenrpcspecs.js]: Awesome!!!

**Pull request merged**